__tests__/app.test.js

```diff
@@ -405,3 +405,28 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/users/:username", () => {
+  test("Status:200, responds with a user object", () => {
+    return request(app)
+      .get("/api/users/icellusedkars")
+      .expect(200)
+      .then((res) => {
+        expect(res.body.user).toEqual({
+          username: "icellusedkars",
+          name: "sam",
+          avatar_url:
+            "https://avatars2.githubusercontent.com/u/24604688?s=460&v=4",
+        });
+      });
+  });
+
+  test("Status:404, responds with error when passed a bad username", () => {
```

Comment:
I think `responds with error when user does not exist` would be better than `bad username` 😄